jetty-annotations/src/main/java/org/eclipse/jetty/annotations/AnnotationParser.java

```diff
@@ -86,48 +85,44 @@
     public static int asmVersion ()
     {
         int asmVersion = ASM_OPCODE_VERSION;
-        Package asm = Opcodes.class.getPackage();
-        if (asm == null)
-            LOG.warn("Unknown asm runtime version, assuming version {}", ASM_OPCODE_VERSION_STR);
+        String version = ManifestUtils.getVersion(Opcodes.class).orElse(null);
+        if (version == null)
+        {
+            LOG.warn("Unknown ASM version, assuming {}", ASM_OPCODE_VERSION);
```

Comment:
No, don't do this: I just specifically put in the ASM_OPCODE_VERSION_STR so it prints out something human readable rather than a long, meaningless integer.

Reply:
Doh, I see now the number is not `6`, will revert.